### PR TITLE
lite: init at 1.03

### PR DESCRIPTION
**Pull request by prusnak**
###### Motivation for this change

Ultrafast editor written in Lua

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/editors/lite/default.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  buildPhase = ''
+    sh build.sh
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin $out/lib/${pname}
+    cp -a lite $out/lib/${pname}
+    cp -a data $out/lib/${pname}
+    makeWrapper $out/lib/${pname}/lite $out/bin/lite
```
Comment (tadeokondrak) [inline pkgs/applications/editors/lite/default.nix:30]: What is this wrapper for?

**Comment by aaronjanse** [comment]
I'd just like to confirm that I've built this PR on my laptop and, as far as I can tell, `lite` works as intended.

**Comment by mmahut** [comment]
Problem on darwin:

```
/nix/store/0np9zfyh5rq4kymr2ldqj5w2n82v1n30-stdenv-darwin/setup: line 1315: gcc: command not found
```

**Comment by prusnak** [comment]
@GrahamcOfBorg build lite

**Comment by prusnak** [comment]
darwin fixed in d90df25

**Comment by prusnak** [comment]
Added @filalex77 as a maintainer as per https://github.com/NixOS/nixpkgs/pull/87589#issuecomment-626948757

**Review by Br1ght0ne — APPROVED** [review]

**Comment by prusnak** [comment]
@GrahamcOfBorg build lite

**Comment by HugoReeves** [comment]
Successfully built and executed the package on NixOS and MacOS Catalina.

Note that the 1.03 version does not have correct scaling on MacOS. [This comment thread](https://github.com/rxi/lite/issues/46#issuecomment-626792172) in the lite repo indicates that supporting MacOS is probably not within the scope of the rxi/lite repo. Maybe setting `platforms = platforms.linux;` until MacOS support is confirmed would be advisable.

**Comment by mmahut** [comment]
@GrahamcOfBorg build lite

**Comment by prusnak** [comment]
@GrahamcOfBorg build lite

**Pull request merged**